resources/views/store/manage_vouchers.blade.php

```diff
@@ -157,38 +157,50 @@
-                            <label for="collected-by">Collected by:</label>
-                            <select id="collected-by">
-                                @foreach( $carers as $carer )
-                                    <option value="{{ $carer->id }}">{{ $carer->name }}</option>
-                                @endforeach
-                            </select>
+                            <i class="fa fa-user"></i>
+                            <div>
+                                <label for="collected-by">Collected by:</label>
+                                <select id="collected-by" name="collected_by">
+                                    @foreach( $carers as $carer )
+                                        <option value="{{ $carer->id }}">{{ $carer->name }}</option>
+                                    @endforeach
+                                </select>
+                            </div>
                         </div>
-                    </div>
-                    <div>
-                        <i class="fa fa-calendar"></i>
                         <div>
-                            <label for="collected-on">Collected on:</label>
-                            <input id="collected-on" value="<?php echo date('Y-m-d');?>" type="date">
+                            <i class="fa fa-calendar"></i>
+                            <div>
+                                <label for="collected-on">Collected on:</label>
+                                <input id="collected-on" name="collected_on" value="<?php echo date('Y-m-d');?>"
+                                       type="date">
+                            </div>
                         </div>
-                    </div>
-                    <div>
-                        <i class="fa fa-home"></i>
                         <div>
-                            <label for="collected-at">Collected at:</label>
-                            <select id="collected-at">
-                                {-- Will need to be a list of all local centres eventually --}
-                                <option value="{{ $centre->id }}">{{ $centre->name }}</option>
-                            </select>
+                            <i class="fa fa-home"></i>
+                            <div>
+                                <label for="collected-at">Collected at:</label>
+                                <select id="collected-at" name="collected_at">
+                                    {-- Will need to be a list of all local centres eventually --}
+                                    <option value="{{ $centre->id }}">{{ $centre->name }}</option>
+                                </select>
+                            </div>
                         </div>
+                        <button class="short-button"
+                                type="submit">
+                            Confirm pick up
+                        </button>
+                        <button class="long-button"
```

Comment:
Don't like having this button in the form - but if semantically allowed - I guess it'll work.